lib/mix/tasks/phx.gen.context.ex

```diff
@@ -99,7 +99,10 @@ defmodule Mix.Tasks.Phx.Gen.Context do
     {opts, parsed, _} = parse_opts(args)
     [context_name, schema_name, plural | schema_args] = validate_args!(parsed)
 
-    table = Keyword.get(opts, :table, Phoenix.Naming.underscore(context_name) <> "_#{plural}")
+    table = Keyword.get(opts, :table, (context_name 
```

Comment:
Done. But I don't know why the Travis CI build failed, maybe it is expecting this:
```elixir
table_name = 
    context_name 
    |> Phoenix.Naming.underscore 
    |> String.replace("/", "_")
```
instead of:
```elixir
table_name = context_name |> Phoenix.Naming.underscore |> String.replace("/", "_")
```

At the job log it crashes within the integration endpoint test
```
test adapters starts on configured port and serves requests and stops for dev (Phoenix.Integration.EndpointTest)
     test/phoenix/integration/endpoint_test.exs:136
     Assertion with =~ failed
```
I'm sorry for the trouble, it's my first contribution.